community/modules/file-system/gke-persistent-volume/main.tf

```diff
@@ -20,12 +20,16 @@ locals {
 }
 
 locals {
-  location             = split("/", var.filestore_id)[3]
-  filestore_name       = split("/", var.filestore_id)[5]
+  is_gcs = (var.gcs_bucket_name != null)
+
+  filestore_id         = !local.is_gcs ? var.filestore_id : "projects/empty/locations/empty/instances/empty"
```

Comment:
nit. Is there a meaning for `"projects/empty/locations/empty/instances/empty"` ?